.gitignore

```diff
@@ -1 +1,7 @@
-node_modules
+/node*
+*.env*
+.eslintrc.js
+.editorconfig
```

Comment:
Config files are auto generated on `make install` so we keep that as the source of truth